### PR TITLE
Finish scheduler

### LINKED ISSUE #21: Agent is triggered to ask start conversation on foreseen HRS's
- At 9:00 AM we ask user whether the user foresees any High-Risk-Situations (HRS) for smoking, answer yes/no
- In case HRS is foreseen we say 'Hey, remember you can deal with cravings by watering your plants' (for now that is fine, later this should come from the database)
- In case of no HRS we say: 'Please be careful, cravings can always come unexpectedly'

- [ ] Add this story to rasa (add intents + utterances)
- [x] Come up with a way to trigger rasa to start this conversation. There's roughly 2 options: (1) From within rasa. (2) In a separate application that sends a POST request to rasa with a certain intent  from a user so that the conversation is started.
- [x] Implement the triggering (done in #41)
- [x] #112 
- [ ] Make sure everything works with docker-compose in the root, this includes enabling custom channel again, see: https://github.com/PerfectFit-project/virtual-coach-server/blob/3e3bf99610386b8803be28c8c377e87f42181559/Rasa_Bot/credentials.yml#L11
NB: In https://github.com/PerfectFit-project/virtual-coach-server/pull/131 we introduced a requirement for rasa server, so we need to define our own dockerfile (like we doe for actions server) to make this work.

[rasa tutorial](https://towardsdatascience.com/chatbots-made-easier-with-rasa-2-0-b999323cdde)

### PR DESCRIPTION
**Pull request by svenvanderburg**
Fixes #21 
* Add conversation on foreseen smoking HRS to rasa
* Add rasa scheduler to main docker-compose application

Branches of `enable-custom-channel` (#140), so merge that first.

### CODE REVIEW

File: Rasa_Bot/tests/test_conversations.yml [hunk 1]
```diff
@@ -13,4 +13,8 @@ stories:
   - user: |
       Nee
     intent: deny
-  - action: utter_okay
\ No newline at end of file
+  - action: utter_okay
+- story: Ask foreseen HRS
+  steps:
+  - intent: EXTERNAL_trigger_ask_foreseen_hrs
+  - action: utter_ask_foreseen_hrs
```
[the remark this answers is not in the document]
Comment (raar1) [reply Rasa_Bot/tests/test_conversations.yml:20]: Weird. Does Rasa actually treat intentions starting with `EXTERNAL_` differently?

[the remark this answers is not in the document]
Comment (raar1) [reply Rasa_Bot/tests/test_conversations.yml:20]: Not sure if it is related to this (old-ish) issue about intent triggering vs action triggering: <https://github.com/RasaHQ/rasa/issues/4464> ?

File: docker-compose.yml [hunk 2]
```diff
@@ -19,4 +21,12 @@ services:
     env_file:
       - .env
     expose: ["8080"]
-
+  redis:
```
Comment (raar1) [inline docker-compose.yml:24]: I suppose this is what is replacing django as ~db?

Comment (svenvanderburg) [reply docker-compose.yml:24]: No, in Cunliang's setup there was also already a redis server, that acts as a broker/message queue (I think, I'm not a celery expert (yet)). The django db was for storing the different tasks.

File: Rasa_Bot/tests/test_conversations.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (svenvanderburg) [reply Rasa_Bot/tests/test_conversations.yml]: I think I made a booboo. I was about to raise an issue at Rasa, so I thought let's double check. And now it seems to work flawlessly... Thanks for having me retry this.

**Pull request merged**